app/Repositories/CircleRepository.php

```diff
@@ -85,4 +90,20 @@ public function getWebhook($circle_id) {
         $circle = $this->model->find($circle_id);
        return $circle->discord_webhook ?:'';
     }
+
+    public function addCoordinapeUserToCircle($circle_id) {
```

Comment:
maybe you could change this to take an array of circle IDs and then bulk insert as Zashton suggested above

Reply:
yeah if you're doing that
move 
`$profile = Profile::firstOrCreate([ 'address' => config('ape.coordinape_user_address') ]);`
this inside the function as well

Reply:
that would make this better, but lets merge this version first.